test/tests/functional/pbs_equiv_classes.py

```diff
@@ -1392,12 +1353,133 @@ def test_incr_decr(self):
                             {'scheduling': 'True'})
 
         # No message for equivalence class
-        try:
-            m = self.scheduler.log_match("Number of job equivalence classes",
-                                         max_attempts=10,
-                                         starttime=int(self.t))
-        except PtlLogMatchError, e:
-            self.logger.info(
-                "Number of job equivalence classes message\
-                not present when there are no jobs as expected")
-            self.assertNotEqual(e.rc, 0)
+        self.scheduler.log_match("Number of job equivalence classes",
+                                 max_attempts=10,
+                                 starttime=self.t,
+                                 existence=False)
+        self.logger.info(
+            "Number of job equivalence classes message" +
+            "not present when there are no jobs as expected")
+
+    def test_server_queue_limit(self):
+        """
+        Test with mix of hard and soft limits
+        on resources for users and groups
+        """
+
+        # Update sched_config
+        a = {'preempt_prio':
+             'express_queue, normal_jobs, server_softlimits,\
+             queue_softlimits,server_softlimits+queue_softlimits'}
+        self.scheduler.set_sched_config(a)
```

Comment:
You don't need to set this unless you really want to.  The only reason preempt_prio needs to be set is if preemption needs to happen.  I don't believe you want any preemption to actually happen during this test.